packages/tasks/src/local-apps.ts

```diff
@@ -0,0 +1,119 @@
+import type { ModelData } from "./model-data";
+import type { PipelineType } from "./pipelines";
+
+/**
+ * Elements configurable by a local app.
+ */
+export type LocalApp = {
+	/**
+	 * Name that appears in buttons
+	 */
+	prettyLabel: string;
```

Comment:
just making a comment that previously we ended up mixing snake_case & camelCase like in https://github.com/ngxson/huggingface.js/blob/5f547dd882ad07dad612225f6b8193845fb318a9/packages/tasks/src/model-data.ts#L125-L143